.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+
+# Controls when the workflow will run
+on:
+    schedule:
+        - cron: '*/15 * * * *'
```

Comment:
Do we need the tests to run on a Cron? The only reason the lint one has that was for a workaround to get those inline code comments working, and even then I'm not 100% sure it was necessary